.gitignore

```diff
@@ -26,6 +26,7 @@ docs/_build
 **/.env
 **/.factorypath
 kubernetes/examples/helm/charts/
+examples/
```

Comment:
Ignoring this dir can cause new quick start resource files being ignored

Reply:
Hmm, I see. Reverted.